client/editEvents.js

```diff
@@ -160,13 +160,13 @@ Template.editEvents.helpers({
     }
     return this.size > maxAllowedRoundSize;
   },
-  roundComplete: function() {
+  progressClasses: function() {
```

Comment:
Looks like good cleanup. It's kind of unfortunate to be mixing the view logic up in this code, but since handlebars is so neutered, I imagine this is a lot cleaner.